zuul.d/tempest_multinode.yaml

```diff
@@ -1,18 +1,20 @@
 ---
 - job:
-    name: cifmw-base-multinode-tempest
+    name: cifmw-base-multinode-master-tempest
     parent: cifmw-podified-multinode-edpm-base-crc
     timeout: 5400
-    abstract: true
     nodeset: centos-9-medium-crc-extracted-2-30-0-3xl
+    description: |
+      Base job definition for running test-operator with master release.
     vars:
       zuul_log_collection: true
       cifmw_deploy_edpm: false
-      openstack_release: antelope
+      openstack_release: main
```

Comment:
but there is no such openstack release. I'll have to grep to understand this some more like where is this consumed/how does it make sense to set it to main.

Reply:
rdo